internal/server/get_documents.go

```diff
@@ -4,42 +4,49 @@ import (
 	"net/http"
 
 	"github.com/go-chi/chi/v5"
-	"github.com/ministryofjustice/opg-sirius-lpa-frontend/internal/sirius"
 
+	"github.com/ministryofjustice/opg-sirius-lpa-frontend/internal/sirius"
 	"github.com/ministryofjustice/opg-go-common/template"
 )
 
 type GetDocumentsClient interface {
-	DigitalLpa(ctx sirius.Context, uid string) (sirius.DigitalLpa, error)
+	CaseSummary(ctx sirius.Context, uid string) (sirius.CaseSummary, error)
 	Documents(ctx sirius.Context, caseType sirius.CaseType, caseId int, docTypes []string, notDocTypes []string) ([]sirius.Document, error)
 }
 
 type getDocumentsData struct {
 	XSRFToken string
 
-	Lpa          sirius.DigitalLpa
+	CaseSummary  sirius.CaseSummary
 	Documents    []sirius.Document
 	FlashMessage FlashNotification
 }
 
 func GetDocuments(client GetDocumentsClient, tmpl template.Template) Handler {
 	return func(w http.ResponseWriter, r *http.Request) error {
+		var err error
+
 		uid := chi.URLParam(r, "uid")
 		ctx := getContext(r)
-		lpa, err := client.DigitalLpa(ctx, uid)
-		if err != nil {
-			return err
+
+		data := getDocumentsData{
+			XSRFToken: ctx.XSRFToken,
 		}
 
-		documents, err := client.Documents(ctx, "lpa", lpa.ID, []string{}, []string{sirius.TypeDraft, sirius.TypePreview})
```

Comment:
Unrelated to this ticket but I wonder why we hardcode to fetch only "lpa" documents? `caseType` can be "lpa", "epa" or "digital_lpa" - surely we want documents of all types 🤷🏼‍♀️

Reply:
I don't think I changed this, but it's a good question.

Reply:
Should we be passing the case type of the case we're displaying documents for? This does seem to work for digital LPAs after documents have been added to it, so I'm not 100% sure what this is for.